doc/news/changes/minor/20191018NiklasFehn

```diff
@@ -0,0 +1,4 @@
+Refactor: improve implementation of Utilities::needed_digits() by using
+functionality from the standard library.
+<br>
+(Niklas Fehn, 2019/10/18)
```

Comment:
It doesn't hurt, but in general we only add changelog entries for things that change user-visible behavior. This one doesn't.